app/package.json

```diff
@@ -3,7 +3,7 @@
   "productName": "GitHub Desktop",
   "bundleID": "com.github.GitHubClient",
   "companyName": "GitHub, Inc.",
-  "version": "1.7.0-beta9",
+  "version": "1.7.1-beta1",
```

Comment:
We had to bump the build version here to `1.7.1` because of how Squirrel.Windows currently handles prerelease builds which are later than `-beta9`:

 - This has been fixed when generating releases in https://github.com/anaisbetts/NuGet/pull/1 and https://github.com/Squirrel/Squirrel.Windows/pull/1353 which are available in Squirrel.Windows 1.9.0
 - The outstanding pull request https://github.com/Squirrel/Squirrel.Windows/pull/1492 to address this in another part of Squirrel.Windows needs to be reviewed and published in a future update

We're looking into upgrading this dependency in https://github.com/desktop/desktop/issues/5197 and https://github.com/desktop/desktop/pull/7647, but we need to be mindful of these prereleases until we can get this fixed upstream...